Software/Development/SemesterProject/System/systemInit.c

```diff
@@ -146,7 +147,8 @@ void systemInit(void)
     initPWM();
     initAnalogSignals();      // Initialize the analog signals and their ADC channels
     initWatchdog();
-
+	initDigitalOutputs();

```

Comment:
why tabs? not big deal

Reply:
I dont think the tab is there in the newest version